packages/adapter-node/src/handler.js

```diff
@@ -51,6 +52,17 @@ await server.init({
 	read: (file) => createReadableStream(`${asset_dir}/${file}`)
 });
 
+/** @type {import('polka').Middleware} */
+const middleware = async (req, res, next) => {
+	const { pathname } = polka_url_parser(req);
+
+	if (pathname.startsWith(`/${manifest.appPath}/immutable/`)) {
+		return next();
+	}
```

Comment:
What's the rationale for this? Even if it seems like most middleware use cases would ignore these assets, is it for us to decide? And why only `/_app/immutable` and not all of `/_app`? Feels like it would be simpler to just let middleware handle all requests, and let users decide which ones to ignore